### ISSUE TITLE
[torch and tf] gs.where

### ISSUE THREAD

**Issue by opeltre** [geomstats/geomstats#399]
Testing the `GL.belongs` method calling:
```
return gs.where(det != 0, True, False)
```
errors in pytorch and tf

[torch]
```
    return torch.where(*args, **kwargs)
TypeError: where(): argument 'input' (position 2) must be Tensor, not bool
```
[tf]
first complains:
```
WARNING:tensorflow:From /home/.local/lib/python3.6/site-packages/tensorflow_core/python/ops/linalg/linalg_impl.py:283: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
```
is this due to python 3.6.10?

then `GL.belongs(a)` silently returns empty tensor of shape `[]`

**Comment by opeltre** [comment]
EDIT: fixed on `torch`, should come with PR #391

**Comment by ninamiolane** [comment]
There is no need to use `gs.where` as `det != 0.` or `~gs.isclose(det, 0.)` can perform this operation. This is maybe the reason why pytorch does not prioritize having `torch.where` working with booleans.
